Update structs.md

lmartelli opened the pull request:
Fix translation.

lmartelli committed:
Update structs.md

Fix translation.

BBasile commented on basics/structs.md:
```diff
@@ -73,7 +73,7 @@ p.faireChosePrivee(); // Interdit
 Si une méthode est déclarée avec `const`, elle ne sera pas autorisée à modifier
 un membre de la structure. C'est une garantie offerte par le compilateur.
-Déclarer une méthode `const` permet de l'appeller sur un objet `const` ou `immutable`, mais garantie également que la méthode ne changera pas l'état de l'objet.
+Déclarer une méthode `const` permet de l'appeller sur un objet `const` ou `immutable`, mais garantie également aux appelant que la méthode ne changera pas l'état de l'objet.
```
appelant__s__ car pluriel par `aux`

schlupa committed:
Correction de fautes d'orthographe

Appeler prend 1 l quand on entend le son eu et deux quand c'est é. Donc appeler, appelé, nous appelons mais j'appelle, ils appellent etc.
Cette faute a été faite systématiquement dans tous les fichiers, ce serait bien que le traducteur apprenne la règle ;-)

schlupa commented on basics/structs.md:
```diff
@@ -73,18 +73,18 @@ p.faireChosePrivee(); // Interdit
 Si une méthode est déclarée avec `const`, elle ne sera pas autorisée à modifier
 un membre de la structure. C'est une garantie offerte par le compilateur.
-Déclarer une méthode `const` permet de l'appeller sur un objet `const` ou `immutable`, mais garantie également que la méthode ne changera pas l'état de l'objet.
+Déclarer une méthode `const` permet de l'appeler sur un objet `const` ou `immutable`, mais garantie également aux appelants que la méthode ne changera pas l'état de l'objet.
```
appelants s'accorde, ce n'est pas un verbe au participe présent mais un substantif.

schlupa commented on basics/structs.md:
```diff
@@ -73,18 +73,18 @@ p.faireChosePrivee(); // Interdit
 Des hiérarchies de types ne peuvent être construites qu'avec des classes,
 que nous verrons dans une prochaine aprtie.
-Cependant, avec `alias this` ou les `mixins`, on peut facilement obtenir du molyporphisme.
+Cependant, avec `alias this` ou les `mixins`, on peut facilement obtenir du polymorphisme.
```
lol
